modules/client/examples/05-encoders-decoders/12-apply-uninstallation.ts

```diff
@@ -27,7 +28,7 @@ const client: Client = new Client(context);
 const applyUninstallationParams: ApplyUninstallationParams = {
   permissions: [{
     condition: "0x1234567890123456789012345678901234567890",
-    operation: 1,
+    operation: PermissionOperationType.GRANT,
```

Comment:
are we granting a permission when uninstalling?

Reply:
true, my bad